frontend/public/components/monitoring/targets.tsx

```diff
@@ -1,26 +1,32 @@
-import * as _ from 'lodash-es';
 import {
   GreenCheckCircleIcon,
+  ListPageBody,
   PrometheusEndpoint,
   RedExclamationCircleIcon,
   RowFilter,
+  RowProps,
+  TableColumn,
 } from '@console/dynamic-plugin-sdk';
-import { Alert, Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import {
+  ListPageFilter,
+  ListPageHeader,
+  ResourceLink,
+  Timestamp,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@console/dynamic-plugin-sdk/src/lib-core';
+import { Alert, Breadcrumb, BreadcrumbItem, Tooltip } from '@patternfly/react-core';
 import { sortable } from '@patternfly/react-table';
+import * as fuzzy from 'fuzzysearch';
+import * as _ from 'lodash-es';
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
 import { useTranslation } from 'react-i18next';
 // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
 // @ts-ignore
```

Comment:
This can be removed as we removed the code causing this error
```suggestion
```

Reply:
👍  Oops